src/utils/documentUtils.ts

```diff
@@ -40,4 +43,44 @@ const getDocPropsProperties = async (zip: JSZip): Promise<{ doc: Document; prope
     return { doc, properties };
 };
 
-export default { createOrUpdateProperty, getDocPropsProperties };
+const getCellReference = (col: number, row: number): string => {
+    return String.fromCharCode(col + A) + "$" + (row).toString();
+} 
+
+const createCell = (doc: Document, parent: Element, colIndex: number, rowIndex: number, dataType: number, data: string): void => {
+    const cell = doc.createElementNS(doc.documentElement.namespaceURI, "c");
+    cell.setAttribute("r", getCellReference(colIndex, rowIndex + 1).replace("$", ''));
+    const cellData = doc.createElementNS(doc.documentElement.namespaceURI, "v");
+    updateCellData(dataType, data , cell, cellData);
+    cell.appendChild(cellData);
+    parent.appendChild(cell);
```

Comment:
parent isn't part of the logic here, it should be managed by the caller, and then your function is testable.